tinkerpop3/src/main/java/io/shiftleft/overflowdb/OdbGraph.java

```diff
@@ -137,10 +138,25 @@ private void initElementCollections(OdbStorage storage) {
     }
 
     currentId.set(maxId + 1);
+    initializeStoredIndices(storage);
     long elapsedMillis = System.currentTimeMillis() - start;
     logger.info("initialized " + this.toString() + " from existing storage in " + elapsedMillis + "ms");
   }
 
+  private void initializeStoredIndices(OdbStorage storage) {
```

Comment:
I introduced the `IndexManager` in the last PR and was hoping that we could move all index related functionality there - this applies to all changes in this file.